package.json

```diff
@@ -216,6 +216,24 @@
                 "command": "azureResourceGroups.showGroupOptions",
                 "title": "%azureResourceGroups.showGroupOptions%",
                 "category": "Azure"
+            },
+            {
+                "command": "azureTenant.refresh",
+                "title": "%azureResourceGroups.refresh%",
+                "category": "Azure",
+                "icon": "$(refresh)"
+            },
+            {
+                "command": "azureTenant.refreshTree",
```

Comment:
What's the difference between this command and the other one?

Reply:
I mostly took this formatting from the way workspace does refresh but I can probably remove one of them.

Reply:
That makes sense, thanks!